.github/dependabot.yml

```diff
@@ -0,0 +1,22 @@
+version: 2
+updates:
+  - package-ecosystem: "pip"
+    directory: "/"
+    schedule:
+      interval: "daily"
+      time: "07:00"
+    versioning-strategy: auto
+    # Allow up to 10 open pull requests for updates to dependency versions
+    open-pull-requests-limit: 10
+    labels:
+      - "part:tooling"
+    reviewers:
+      - "frequenz-floss/python-sdk-team"
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```

Comment:
Why one weekly and the other daily?

Reply:
I don't have a specific reason for it, but I think that checking the versions of  Github Actions on a weekly basis would be enough.
Since we have many pip dependencies, Dependabot creates a pull request for every version update. By default, only up to 5 PRs can be open at a time (although I've adjusted the configuration to allow up to 10). If we update the dependencies versions on a daily basis, it may help ensure that they stay up-to-date in case we reach the PR limit (and some PRs get merged).

Reply:
These can also have security issues, which could leak credentials for example. At least for security issues I would check it daily.

Reply:
That's a good point, `daily` it is then.